src/Components/src/Microsoft.AspNetCore.Components.Server/Builder/BlazorApplicationBuilderExtensions.cs

```diff
@@ -87,12 +87,6 @@ public static class BlazorApplicationBuilderExtensions
                 });
             }
 
-            // Accept debugger connections
-            if (config.EnableDebugging)
```

Comment:
Is there a plan/need to clean this up in BlazorConfig? Or is this a temporary thing while we figure out how to ship all of this. 

Personally, I feel like having this show up in startup code is an improvement because it's more inline with our other diagnostics features.

Reply:
Yes, I'm sure we'll review and change the Blazor build/hosting mechanisms as we head towards a stable release of that.